src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -14,242 +14,167 @@
 import socket
 import re
 
```

Comment:
I'm not sure it can cause any problems, but I noticed the file permissions for this file changed from 0755 to 0644.

Reply:
It should not be a problem because this is called by another python script